ksg_nett/settings.py

```diff
@@ -163,3 +163,9 @@
 
 STATIC_ROOT = 'static/'
 MEDIA_ROOT = 'media/'
+
+# GLOBAL SETTINGS
+# ------------------------------
+MINIMUM_SOCI_AMOUNT = 0
+SOCI_CARD_ID = 8008135  # For identifying Soci's own Soci Bank Account (a normal card id is 9 digits)
```

Comment:
I believe it is at least 10 (with one or more trailing zeros). We should keep in mind that this _could_ be wrong in the future. I also suspect that there could exist cards with 3 or more number of trailing zeros. Hence changing the value to something more esoteric might be of interest.

Reply:
OK. In this case I aimed for an id that wouldn't collide with an arbitrary, real number, and also had a funny 1337-meaning

Reply:
I wonder if all RFID card UIDs are 8 bytes, in which case something like "8008135101" would work.

Reply:
Nevermind, it seems to vary a bit from vendor to vendor. I suppose Sikringskomitteen would the guys to ask regarding the cards we use?

Reply:
I feel like having a value larger than 8 bytes is safer. I am not convinced cards with 8-byte UIDs ubiquitously have 10 digits with no leading zeros. A value larger than 8 bytes will never collide, however.